Draft-Accepted/RFCNNNN-Install-PowerShell.md

```diff
@@ -0,0 +1,218 @@
+---
+RFC: RFC00XX
+Author: Jason Helmick
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Core
+Comments Due: 6/30/2022
+Plan to implement: Yes
+---
+
+# Install-PowerShell7 Cmdlet for Windows
+
+This RFC proposes the addition of the `Install-PowerShell7` cmdlet and new `PowerShell7` module for
```

Comment:
```suggestion
This RFC proposes is the addition of the `Install-PowerShell7` cmdlet and new `PowerShell7` module for
```

Reply:
I think the original reads better - so not accepting this suggestion

Reply:
My bas, I read `proposes` as `process` :)